docs/UserGuide.adoc

```diff
@@ -101,19 +101,34 @@ Example(s):
 //end::addmeds[]
 //tag::addmh[]
 === Add to patient’s medical history: `addmh`
-Add an entry to an existing patient’s medical history.
+Add an non-blank diagnosis entry to an existing patient’s medical history.
 The patient must be registered within the system.
 
 Format: `addmh ic/NRIC mh/DIAGNOSIS​ doc/DOCTOR-IN-CHARGE`
 ****
-* If the NRIC does not match to an existing patient in the system, an error will be displayed.
-* If the format of the doctor's name is incorrect, an error will be displayed.
+WARNING: The following invalid inputs will result in a command failure, and the display of an appropriate error message.
+
+* Invalid NRIC
+** The patient NRIC does not match to an existing patient in the system. The person will first need to be registered.
+* Incorrect format of the doctor's name.
+** Doctor's title must be included.
+** The first letter of all words in doctor's name must be captalised.
+* Blank Diagnosis
```

Comment:
Just wondering if Diagnosis should or should not be capitalised here

Reply:
It shouldn't be capitalised, thanks for pointing out.